dev-python/pendulum/pendulum-2.1.2.ebuild

```diff
@@ -21,7 +21,7 @@ KEYWORDS="~amd64 ~x86"
 # commented for future tests fixups
 RESTRICT="test"
 
-RDEPEND="
+DEPEND="
```

Comment:
I'll revbump anyway because it's cheap for a Python package

Reply:
Thanks, I forgot to revbump